README.md

```diff
@@ -133,6 +133,9 @@ Here's a table with summary of supported `react-native` versions when Fabric is
 
 ## Usage with [react-navigation](https://github.com/react-navigation/react-navigation)
 
+> [!CAUTION]
+> NativeStack has been moved from react-native-screens/native-stack to @react-navigation/native since version v6. With react-native-screens v4 native stack v5 (react-native-screens/native-stack) is deprecated and marked for removal in the upcoming minor release, react-native-screens will support only @react-navigation/native-stack v7.
```

Comment:
```suggestion
> [!CAUTION]
> NativeStack has been moved from react-native-screens/native-stack to @react-navigation/native since version v6. With react-native-screens v4 native stack v5 (react-native-screens/native-stack) is deprecated and marked for removal in the upcoming minor release, react-native-screens v4 will support only @react-navigation/native-stack v7.
```

Reply:
Changed in: https://github.com/software-mansion/react-native-screens/pull/2373/commits/19532d8e6a3b0dd5128f1ad51b4928933b923a26